### PR TITLE
fix copied errors and wording

### LINKED ISSUE #5: Goerli Nethermind Include explanation on the two endpoints: RPC and Engine API
Goerli Nethermind Include explanation on the two endpoints: RPC and Engine API
detailed in dappnode/DAppNode#461 
1/2 of dappnode/DAppNode#464

### PR DESCRIPTION
**Pull request by alexpeterson91**
Fixed the RPC API to reflect proper port. 
edited wording to match goerli geth. 
fixed reference to geth

will close #5

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Goerli Nethermind (Execution Client)
-Goerli is ready for the merge!
+Goerli is now ready for the merge!
 
-There are now two APIs in the Geth Execution Client:
+There are now two APIs in Execution Clients:
 
-1. RPC API `http://nethermind-goerli.public.dappnode:8551`. Use this one to query transactions on your node and connect to your web3 wallet.
-2. Engine API `http://nethermind-goerli.public.dappnode:8551`. Use this one to connect your Beacon Chain (Consensus Layer) client.
+1. RPC API `http://nethermind-goerli.public.dappnode:8545`. Use this endpoint to query transactions on your node and connect to your web3 wallet.
```
Comment (alexpeterson91) [inline getting-started.md]: Was wondering if this should be called RPC API because the API engine is also an RPC API.   Shouldn't this be Querying-API?

Comment (alexpeterson91) [reply getting-started.md]:
```suggestion
1. Querying API `http://nethermind-goerli.public.dappnode:8545`. Use this endpoint to query transactions on your node and connect to your web3 wallet.
```

File: getting-started.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Goerli Nethermind (Execution Client)
-Goerli is ready for the merge!
+Goerli is now ready for the merge!
 
-There are now two APIs in the Geth Execution Client:
+There are now two APIs in Execution Clients:
```
Comment (alexpeterson91) [inline getting-started.md]:
```suggestion
There are now two RPC APIs in Execution Clients:
```

File: getting-started.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-1. RPC API `http://nethermind-goerli.public.dappnode:8551`. Use this one to query transactions on your node and connect to your web3 wallet.
-2. Engine API `http://nethermind-goerli.public.dappnode:8551`. Use this one to connect your Beacon Chain (Consensus Layer) client.
+1. RPC API `http://nethermind-goerli.public.dappnode:8545`. Use this endpoint to query transactions on your node and connect to your web3 wallet.
+2. Engine API `http://nethermind-goerli.public.dappnode:8551`. Use this endpoint to connect your Beacon Chain (Consensus Layer) client.
 
 After the merge, if your Execution Client is not connected to a Consensus Layer client, you won't be able to use it to query the blockchain nor to connect it to your wallet!
```
Comment (alexpeterson91) [inline getting-started.md]:
```suggestion
After the merge, if your Execution Client is not connected to a Consensus Layer client, you won't be able to use it to query the blockchain, nor will you be able to connect your wallet to it!
```
grammar

**Review by alexpeterson91 — COMMENTED** [review]
Let me know what you think.
 this is technically accurate, while the existing terminology was not.

**Review by pablomendezroyo — COMMENTED** [review]
I think we must define the metadata files (getting-started, README, setup-wizard..) for each EL and CL in gnosis prater and mainnet. These files will be almost identical. Then we should go repo by repo updating it

**Comment by alexpeterson91** [comment]
Ok. Another issue should be made then with a task for each client.

**Pull request merged**